pom.xml

```diff
@@ -0,0 +1,287 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jqassistant.contrib.common</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.8.0</version>
+  </parent>
+
+  <groupId>org.jqassistant.contrib.plugin</groupId>
+  <artifactId>jqassistant-hcl-plugin</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <name>jQAssistant HCL plugin</name>
+  <url>https://github.com/jqassistant-contrib/jqassistant-hcl-plugin</url>
+
+  <developers>
+    <developer>
+      <id>matthias-kay</id>
+      <name>Matthias Kay</name>
+      <email>github@matthiaskay.de</email>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:jqassistant-contrib/jqassistant-hcl-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jqassistant-contrib/jqassistant-hcl-plugin.git</developerConnection>
+    <url>https://github.com/jqassistant-contrib/jqassistant-hcl-plugin</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <downloadSources>true</downloadSources>
+    <downloadJavadocs>true</downloadJavadocs>
+
+    <jqassistant.version>1.8.0</jqassistant.version>
+  </properties>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.antlr</groupId>
+          <artifactId>antlr4-maven-plugin</artifactId>
+          <version>4.8-1</version>
+        </plugin>
+        <plugin>
+          <groupId>com.buschmais.jqassistant</groupId>
+          <artifactId>jqassistant-maven-plugin</artifactId>
+          <version>${jqassistant.version}</version>
+          <executions>
+            <execution>
+              <id>default</id>
+              <goals>
+                <goal>scan</goal>
+                <goal>analyze</goal>
+              </goals>
+              <configuration>
+                <warnOnSeverity>INFO</warnOnSeverity>
+                <failOnSeverity>MAJOR</failOnSeverity>
+              </configuration>
+            </execution>
+          </executions>
+          <dependencies>
+            <dependency>
+              <groupId>org.jqassistant.contrib.plugin</groupId>
+              <artifactId>jqassistant-hcl-plugin</artifactId>
+              <version>1.0.0-SNAPSHOT</version>
+            </dependency>
+          </dependencies>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
+    <plugins>
+      <!-- download the HCL grammar into out source tree -->
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <version>1.4.0</version>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
```

Comment:
For now it is fine to fetch the grammar at build time, but in the long run this introduces a dependency to an external resource, which is not under your control. The build would break as soons as the structure of the repository changes or the grammar will be removed. 

My suggestion is to copy the grammar to this repository and also to add a readme about the origin of grammar.

Reply:
👌🏻